polyjuice/polyjuice.py

```diff
@@ -192,12 +197,16 @@ def main(args):
         ask_hermione(out_dir)
         log_path = os.path.join(out_dir, 'log.txt')
         log = Lumberjack(log_path, verbose)
-        browse_restricted_section(parent_file, out_dir, zip_dir, modifications, id_pairs, log)
 
-    if zip_dir:
```

Comment:
there appear to be a lot of changes here that have nothing do to with getting the metadata. I dont want to approve those as I don't know about them nor why they were made.